docs: add marketplace commands, capitalize Mia-Platform related terms (#98)
mia-platform/miactl: docs/30_commands.md

#### Synopsis

Delete a single Marketplace item by its ID

```
miactl marketplace delete resource-id [flags]
```

### apply

Create or update Marketplace items

#### Synopsis

Create or update one or more Marketplace items.

You can either specify: - one or more files, with the flag -f - one or more directories, with the flag -d
Supported formats are JSON (.json files) and YAML (.yaml or .yml files).

```
miactl marketplace apply { { -f file-path }... | { -d directory-path }... } [flags]
```

#### Examples

```

# Apply the configuration of the file myFantasticGoTemplate.json located in the current directory to the Marketplace
miactl marketplace apply -f myFantasticGoTemplate.json

# Apply the configurations in myFantasticGoTemplate.json and myFantasticNodeTemplate.yml to the Marketplace, with relative paths
miactl marketplace apply -f ./path/to/myFantasticGoTemplate.json -f ./path/to/myFantasticNodeTemplate.yml

# Apply all the valid configuration files in the directory myFantasticGoTemplates to the Marketplace
miactl marketplace apply -d myFantasticGoTemplates
```

#### Options

```
  -d, --directory string   a path to a directory containing Marketplace resource files

>   -f, --file stringArray   a path to a JSON or YAML file containing a Marketplace resource

Reviewer: @malta895, are these options still correct, or is an update needed?

Author: Yes an update is needed. I'll integrate it with the image upload functionality, as soon as it is finished

```

### list

List Marketplace items

#### Synopsis

List the Marketplace items that the current user can access.

```
miactl marketplace list [flags]
```

### get

Get a Marketplace item

#### Synopsis

Get a single Marketplace item by its ID

```
miactl marketplace get resource-id [flags]
```

### delete

Delete a Marketplace item

#### Synopsis

Delete a single Marketplace item by its ID

```
miactl marketplace delete resource-id [flags]
```

### apply

Create or update Marketplace items

#### Synopsis

Create or update one or more Marketplace items.

The flag -f accepts either files or directories. In case of directories, it explores them recursively.

Supported formats are JSON (.json files) and YAML (.yaml or .yml files).

The file can contain an image object with the following format:
"image": {
	"localPath": "./someImage.png"
}
The localPath can be absolute or relative to the file location.
The image will be uploaded along with the Marketplace item.
Before being applied, the "image" key will be replaced with the "imageUrl" referring to the uploaded image.
You can retrieve the updated item with the "get" command.

> You can also specify the "supportedByImage" in a similar fashion.

Reviewer 2: ```suggestion
You can also specify the "supportedByImage" in a similar way.
```

> Be aware that the presence of both "image" and "imageUrl" and of both "supportedByImage" and "supportedByImageUrl" is illegal.

Author: ```suggestion
Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is illegal.
```

> Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is illegal.

Reviewer 2: may opt for "is not allowed" instead

Author: I wanted to emphasize the fact that doing it raises an error. Maybe we can do this?
```suggestion
Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is ambiguous and raises an error .
```
